### PR TITLE
Fix 2 bugs and add 2 small features

### PR DESCRIPTION
**Pull request by mhxw**
1、fix the 'docker logs -f ethermintdnode0' command
2、fix makefile error caused by DETECTED_OS
3、add the `make localnet-clean` command about clean testnet
4、add the `make localnet-unsafe-reset` command about reset testnet

### CODE REVIEW

**Comment by mhxw** [comment]
@araskachoi , fedekunze requested a review from araskachoi

File: scripts/start.sh [hunk 1]
```diff
@@ -2,4 +2,4 @@
 ethermintd --home /ethermint/node$ID/ethermintd/ start > ethermintd.log &
 sleep 5
 ethermintcli rest-server --laddr "tcp://localhost:8545" --chain-id 7305661614933169792 --trace > ethermintcli.log &
-tail -f /dev/null
\ No newline at end of file
+tail -f ethermintd.log
```
Comment (araskachoi) [inline scripts/start.sh:5]: This should be `/dev/null`. this is a workaround way to keep the script running and redirect output. user can tail manually to view the output.

**Review by araskachoi — CHANGES_REQUESTED** [review]

File: scripts/start.sh (hunk 1 above)
Comment (mhxw) [reply scripts/start.sh:5]: OK, I see that the command line `docker logs -f ethermintdnode0` mentioned in the [document](https://docs.ethermint.zone/quickstart/testnet.html) can't work, so I'll make a change.

Comment (mhxw) [reply scripts/start.sh:5]: @araskachoi  I have changed. https://github.com/ChainSafe/ethermint/pull/448

**Pull request closed without merging**